lib/game_box_web/live/home_live.ex

```diff
@@ -8,35 +8,45 @@ defmodule GameBoxWeb.HomeLive do
   def render(assigns) do
     ~H"""
     <div>
-      <h2>Start or Join an Arena</h2>
-      <form id="join_arena" phx-submit="join_arena">
+      <.h2>Start or Join an Arena</.h2>
+      <form id="join_arena" phx-submit="join_arena" class="flex flex-col w-1/4">
         <div>
-          <label for="player_name">Name</label>
-          <input
+          <.input
+            label="Name"
             type="text"
             id="player_name"
             name="player_name"
             placeholder="Enter user name"
+            class="w-full"
+            value={@player_name}
+            errors={["required"]}
           />
         </div>
         <div>
-          <label for="arena_id">Arena Code</label>
-          <input
+          <.input
+            label="Arena Code"
             type="text"
             id="arena_id"
             name="arena_id"
             placeholder="4 character arena code"
+            class="w-full"
+            value=""
+            errors={["required"]}
           />
         </div>
-
-        <button type="submit">Join Arena</button>
+        <.button color="primary" type="submit">Join arena</.button>
       </form>
     </div>
     """
   end
 
   @impl true
   def mount(_params, session, socket) do
+    socket =
+      socket
+      |> assign_new(:player_name, fn -> "" end)
```

Comment:
I don't see these getting used so I'm wondering do we need these assigns?

Reply:
You're right, I had done this when I was having issues with the values but it turns out I didn't actually need them.

Reply:
My bad, I forgot that I added that whole snippet and just got rid of my fields. Fixed for real this time.